prow/cmd/deck/main.go

```diff
@@ -1290,12 +1290,9 @@ func canTriggerJob(user string, pj prowapi.ProwJob, cfg *prowapi.RerunAuthConfig
 		return false, nil
 	}
 
-	// If the job is a presubmit and has an associated PR, do the same checks as for /test
-	if pj.Spec.Type == prowapi.PresubmitJob && pj.Spec.Refs != nil && len(pj.Spec.Refs.Pulls) > 0 {
-		if pluginAgent == nil {
-			// If no plugins configuration is provided, skip the checks
-			return false, nil
-		}
+	// If the job is a presubmit and has an associated PR, and a plugin config is provided,
+	// do the same checks as for /test
+	if pj.Spec.Type == prowapi.PresubmitJob && pj.Spec.Refs != nil && len(pj.Spec.Refs.Pulls) > 0 && pluginAgent != nil{
```

Comment:
How is this different?

Reply:
It's supposed to be the same. No need for the inner if statement

Reply:
Makes sense -- I was scratching my head trying to figure it out lol